sdk/core/Azure.Core.TestFramework/src/HeaderRegexSanitizer.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.TestFramework.Models
+{
+    public partial class HeaderRegexSanitizer
+    {
+        public static HeaderRegexSanitizer CreateWithQueryParameter(string headerKey, string queryParameter, string value) =>
```

Comment:
Let's add some tests for this and the new one in UriRegExSanitizer.

Reply:
This ended up moving into another PR.